pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -16,18 +16,24 @@
     patches = [];
   }))
 }:
-stdenv.mkDerivation {
+
+let
+  bin_folder = if stdenv.isDarwin then "darwin" else "linux";
+in
+
+stdenv.mkDerivation rec {
   inherit src version;
   pname = "bs-platform";
+
   BS_RELEASE_BUILD = "true";
+  BS_TRAVIS_CI = "1";
```

Comment:
Can you elaborate on why we need this?

Reply:
Thanks for explanation! I would probably add some short comment about this as it's really non obvious from code itself. Thanks